cmake/small/dev-options.cmake

```diff
@@ -0,0 +1,90 @@
+#######################################################
+### Developer mode                                  ###
+#######################################################
+# Developer mode enables targets and code paths in the CMake scripts that are
+# only relevant for the developer(s) of small.
+# Targets necessary to build the project must be provided unconditionally, so
+# consumers can trivially build and package the project
+if (MASTER_PROJECT)
```

Comment:
Should this be `SMALL_MASTER_PROJECT`? Or `SMALL_ROOT_PROJECT`?

Reply:
I guess it should be ``SMALL_MASTER_PROJECT`` Let me update.